packages/fixie/src/serve-bin.ts

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env node
+
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { fastify, FastifyInstance, FastifyRequest, FastifyReply } from 'fastify';
+import { ReadableStream } from 'stream/web';
+import index from './index.js';
+
+async function sendReadableStreamToFastifyReply(reply: FastifyReply, readableStream: ReadableStream) {
+  const reader = readableStream.getReader();
+  // eslint-disable-next-line no-constant-condition
+  while (true) {
+    const { done, value } = await reader.read();
+    if (done) {
+      reply.raw.end();
+      break;
+    }
+    reply.raw.write(value);
+  }
+}
+
+async function serve({
+  packagePath,
+  port,
+  silentStartup,
+}: {
+  packagePath: string;
+  port: number;
+  silentStartup: boolean;
+}): Promise<void> {
+  //const handler = await import(packagePath);
+  const handler = index;
+  const app: FastifyInstance = fastify();
+  app.post('/', async (req: FastifyRequest, res: FastifyReply) => {
+    const body = req.body as { message: { text: string } };
+    try {
+      const responseStream = handler(body.message.text);
+      await sendReadableStreamToFastifyReply(res, responseStream);
```

Comment:
From some poking around I did previously I think you can just write `res.send(responseStream)` -- did that not work?

Reply:
cribbed this from an example - didn't try simplifying yet, will give it a shot